bucket/python.json

```diff
@@ -1,34 +1,40 @@
 {
     "homepage": "https://www.python.org/",
-    "license": "https://docs.python.org/3/license.html",
+    "description": "A programming language that lets you work quickly and integrate systems more effectively.",
+    "license": "Python-2.0",
     "version": "3.7.3",
+    "depends": "dark",
```

Comment:
We could open discussion if prefixing main bucket requirements is suitable. When i am thinking about this, i do not know how current implementation will handle multiple manifests with same name.

For Examle `main/dark`, 'Ash258/dark' If dark is in both buckets, mine will be used (alphabetic order, as far my testing with simple `scoop install Wavebox`, `Ash258` bucket is preferred over `extras`, which contains Wavebox too)